src/ert/run_models/evaluate_parameters.py

```diff
@@ -0,0 +1,78 @@
+from __future__ import annotations
+
+from typing import TYPE_CHECKING
+
+import numpy as np
+
+from ert.ensemble_evaluator import EvaluatorServerConfig
+from ert.run_context import RunContext
+from ert.storage import EnsembleAccessor, StorageAccessor
+
+from . import EnsembleExperiment
+
+if TYPE_CHECKING:
+    from ert.config import ErtConfig, QueueConfig
+    from ert.run_models.run_arguments import (
+        EvaluateParametersRunArguments,
+    )
+
+
+# pylint: disable=too-many-arguments
+class EvaluateParameters(EnsembleExperiment):
```

Comment:
Could we here inherit from base run model instead?

Reply:
Yes, but either need to reimplement that check run-path function or split it out.